src/components/Authentication/Login/Login.jsx

```diff
@@ -0,0 +1,45 @@
+import loginSignupImage from "../../../assets/login-signup.png";
+import "@fontsource/dm-sans";
+
+function Login() {
+  return (
+    <div className="login flex h-screen font-sans">
+      <div className="w-3/5 bg-white text-[#1E0E62] p-4">
+        <p className="text-7xl text-left mt-[99px] ml-[115px] font-bold">
+          Sign in and let&apos;s start your journey!
+        </p>
+        <div className="w-[470px] h-[324px] mt-[51px] ml-[115px]">
+          <div className="email text-lg text-left mt-6">
+            <label className="block font-medium">EMAIL</label>
+            <input
+              type="text"
+              id="email"
+              placeholder="Your email"
+              className="border border-gray-300 w-[470px] h-[60px] rounded-full p-2 mt-[15px] pl-6"
+            />
+          </div>
+          <div className="password text-lg text-left mt-6">
+            <label className="block font-medium">PASSWORD</label>
+            <input
+              type="password"
+              id="password"
+              placeholder="Your password"
+              className="border border-gray-300 rounded-full w-[470px] h-[60px] p-2 mt-[15px] pl-6"
+            />
+          </div>
+          <button
+            type="button"
+            className="bg-[#482BE7] text-white font-bold rounded-full w-[210px] h-[60px] p-2 mt-16 left-[135px] absolute"
+          >
+            Sign In
+          </button>
```

Comment:
this is a sign in button without onClick
It's type is button but I suppose it's function is to be a submit button. ?

Reply:
Changed it's type to "submit" in the new commit.